CONTRIBUTING.md

```diff
@@ -82,4 +82,4 @@ The code for the documentation and the website are located in the meanjs/mean re
 
 For those who will be generating releases:
 * Don't forget to update the changelog!
-  * `conventional-changelog -r 1`
+  * `conventional-changelog -r 1 | cat > CHANGELOG.md`
```

Comment:
If -r 1 is only adding latest version information then we might want to append to the CHANGELOG's top of the file, not overwrite it.

@ilanbiala I assume we want CHANGELOG to have all milestones and not just the latest?